Automattic/simplenote-ios
# Add metadata and translation downloader

This PR adds some tools to download the translations for the app strings and for the metadata.
It also adds the current translations.

### To Test
1. Run `./Scripts/update-translations.rb` and verify that the tool downloads the latest app strings translations from GlotPress (Note that the translations are probably already at the latest version, so you may want to modify some .lproj files before running the script to verify that it works).
2. Run `./download_metadata.swift` and verify that  the metadata translations are downloaded into the `fastlane/metadata` folder Note that the translations are probably already at the latest version, so you may want to modify/delete some files before running the script to verify that it works).
